.github/workflows/daily.yml

```diff
@@ -143,6 +143,31 @@ jobs:
         if: ${{ failure() }}
       - uses: ./.github/actions/build-teardown
 
+  prover-inconsistency-test:
+    runs-on: ubuntu-latest-xl
+    container:
+      image: diem/build_environment:main
+    env:
+      MESSAGE_PAYLOAD_FILE: /tmp/message
+    strategy:
+      fail-fast: false
+      matrix:
+        release-branch: [release-1.1]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ matrix.release-branch }}
+      - uses: ./.github/actions/build-setup
+      - name: Run Prover test with inconsistency check
+        run: |
+          MVP_TEST_INCONSISTENCY=1 cargo test -p move-prover --release
```

Comment:
you should include the sccache creds, but not strictly necessary as some other jobs will likely build them.

Reply:
I added `SCCACHE_AWS_...`s to `env`. Does this look ok?

Reply:
By the way, will it be automatically in the proper working directory? Or, does it need something like `$pre_command && ...` to go to `/opt/git/diem`?

Reply:
ahh yes you do need the precommand.